vsock_sample/py/vsock-sample.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Should it be changed to 2021 here and in the other two places?

Reply:
Yes. Right :)